packages/obj-logger/src/obj-logger.ts

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable no-console */
+import { DataStream, StringStream } from "scramjet";
+import { PassThrough, Writable } from "stream";
+import { IObjectLogger, LogEntry, LogLevel } from "@scramjet/types";
+import { getName } from "./utils/get-name";
+
+export class ObjLogger implements IObjectLogger {
+    outLogStream = new PassThrough({ objectMode: true });
+    inLogStream = new PassThrough({ objectMode: true });
+    inStringStream = new PassThrough({ objectMode: false });
```

Comment:
I think we chatted about it but it escaped me, can you please remind why it was introduced?

Reply:
i will add comments to this class :)